src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -43,6 +43,8 @@ interface Props<S>
     /** Means to overwrite the look and feel of the UI in its blurred state.
      * It should be null or undefined if there is no selection yet */
     customSelectionIndicator?: ReactNode;
+    /** Additional HTML attributes to be applied to the input element */
+    inputAttr?: DictionaryOf<string | boolean>;
```

Comment:
What do you think about renaming `inputAttr`  -> `inputProps`. It will be a more React-friendly way I think.

Reply:
I got your point. What about adding `s` `inputAttrs` ?